silhouette/app/com/mohiva/play/silhouette/impl/providers/state/CsrfStateItemHandler.scala

```diff
@@ -108,7 +109,9 @@ class CsrfStateItemHandler @Inject() (
     item.id == ID && {
       clientState match {
         case Success(i) => i == item.data.as[Item]
-        case Failure(_) => false
+        case Failure(e) =>
+          logger.warn(e.getMessage)
```

Comment:
Could you please add the exception also to the logger, so that the user can see the stack trace in the log.

Reply:
Done